src/System.Memory/tests/Memory/Trim.cs

```diff
@@ -2,10 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Collections.Generic;
 using Xunit;
 
+#pragma warning disable xUnit1025 // analyzer is flagging unique InlineDatas as duplicates
```

Comment:
Do you recall which ones? I would be curios to see what's throwing off the analyzer.

Reply:
Just remove this and you'll see. ;)